arr/values.yaml

```diff
@@ -0,0 +1,131 @@
+# Default values for arr.
+
+image:
+  organization: itscontained
+  repository: ""
+  pullPolicy: IfNotPresent
+  tag: ""
+
+# Probes configuration
+probes:
+  liveness:
+    initialDelaySeconds: 60
+    failureThreshold: 5
+    timeoutSeconds: 10
+  readiness:
+    initialDelaySeconds: 60
+    failureThreshold: 5
+    timeoutSeconds: 10
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+env:
+  TZ: UTC
+
+service:
+  type: ClusterIP
+  port: ""
+  ## Specify the nodePort value for the LoadBalancer and NodePort service types.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+  ##
+  # nodePort:
+  ## Provide any additional annotations which may be required. This can be used to
+  ## set the LoadBalancer service type to internal only.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+  ##
+  annotations: {}
+  labels: {}
+  additionalSpec: {}
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  labels: {}
+  path: /
+  hosts:
+    - chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+persistence:
+  # type: options are statefulset or deployment
+  type: statefulset
```

Comment:
Should this be nested under persistence? It has a huge impact.

Reply:
I put it there only because big charts use persistence as the location as well. [Grafana Example](https://github.com/helm/charts/blob/master/stable/grafana/templates/statefulset.yaml#L1)